src/opal.c

```diff
@@ -739,7 +740,7 @@ binary_unary (char compound_char, lexeme_type_e compound_type,
     if (next_char == EOF)
     {
         /// Illegal character found.
-        sprintf (perror_msg, "Found an illegal character while extracting lexemes on line %s col %s.", char_line, char_col);
+        sprintf (perror_msg, "Found an illegal character while extracting lexemes on line %d col %d.", char_line, char_col);
```

Comment:
Thanks for catching the modifiers. Can you also shorten it to something inline with [GNU standards error message](https://www.gnu.org/prep/standards/standards.html#Errors).